eda-env/Lib/site-packages/pandas/tests/window/test_ewm.py

```
@@ -0,0 +1,727 @@
+from pandas import (
+    DataFrame,
+    DatetimeIndex,
+    Series,
+    date_range,
+)
+import pandas._testing as tm
+
+
+def test_doc_string():
+    df = DataFrame({"B": [0, 1, 2, np.nan, 4]})
+    df
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)